### PR TITLE
Changes in the proposal template + meeting template

### PR DESCRIPTION
**Pull request by RuiAAPeres**
- Added a checkbox, so the author creates an invite for the whole team as a sync point.
- Added the invitation template

### CODE REVIEW

File: TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+This meeting is to discuss the "<Proposal Title>" proposal authored by <Proposal's Author>.
+
+1.	PR here <Link to the Proposal's PR>.
+2.	Actual proposal here <Link to the Proposal's .md file>.
+
+Meeting Etiquette:
+
+1.	**Please fully read the proposal.**
+  a.	If you haven’t read the proposal, or don’t care for the outcome, please don’t show up for the meeting.
+2.	If you want to counter purpose, or raise a change, please provide those in written to all meetings participants beforehand.
```
Comment (mluisbrown) [inline TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md]:
```suggestion
2.	If you want to make a counter proposal, or raise a change, please provide those in written to all meetings participants beforehand.
```

File: TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  a.	Ideally this is done via a PR’s comment.
```
Comment (mluisbrown) [inline TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md]:
```suggestion
  a.	Ideally this is done via a PR comment.
```

File: TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  b.	Meetings notes (or PR's comments) can only be provided 2h before the meeting.
```
Comment (mluisbrown) [inline TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md]:
```suggestion
  b.	Meetings notes (or PR comments) can only be provided 2h before the meeting.
```

File: TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+3.	Questions should be asked on the PR, not during the meeting.
+4.	**The purpose of the meeting is to act as a sync point and a final outcome for the proposal itself.**
+  a. Comments can be made, but at this point, the author should have a good idea if the proposal is to move forward or not.
```
Comment (mluisbrown) [inline TechnicalDocuments/Proposals/Template_Proposal_Meeting_Invitation.md]:
```suggestion
  a. Comments can be made, but at this point, the author should have a good idea if the proposal is going to move forward or not.
```

File: TechnicalDocuments/Proposals/Template_Proposal.md [hunk 5]
```diff
@@ -17,12 +17,13 @@
...
-// This section should explain, assuming this proposal is accepted, how much effort it would require for it to be implemented in our codebase. Other concerns should be raised, if it's a significant deviation from our stack. 
+// This section should explain, assuming this proposal is accepted, how much effort it would require for it to be implemented in our codebase. Other concerns should be raised, if it's a significant deviation from our stack.
 
 ## Alternatives considered
 
 // This section describes what other approaches were considered and why this one was chosen.
 
---- 
+---
+* [ ] I will send a meeting invitation, using this [template](Template_Proposal_Meeting_Invitation.MD), scheduled for 2 weeks after this proposal is made, so an agreement can be reached.
```
Comment (ilyapuchka) [inline TechnicalDocuments/Proposals/Template_Proposal.md]: maybe change this to "I sent a meeting invitation"?

Comment (RuiAAPeres) [reply TechnicalDocuments/Proposals/Template_Proposal.md]: You can't send a meeting invitation before the PR is made, so the invitation will happen in the immediate future after the PR is created.

**Pull request merged**